doc/releases/changelog-dev.md

```diff
@@ -19,6 +19,9 @@
 
 <h3>Breaking changes 💔</h3>
 
+* The `KerasLayer` in `qml.qnn.keras` has been deprecated because Keras 2 is no longer actively maintained.  Please consider using a different machine learning framework.
```

Comment:
```suggestion
* The `KerasLayer` in `qml.qnn.keras` has been removed because Keras 2 is no longer actively maintained.  Please consider using a different machine learning framework.
```